src/services/at-sce-api.service.js

```diff
@@ -40,8 +41,16 @@ AT_SCE_SERVICE.getAllUsers = () => {
 };
 
 // Operation: Get USER by ID - GET/api/v1/users/{id}
-AT_SCE_SERVICE.getUserById = () => {
-  return {};
+AT_SCE_SERVICE.getUserById = (id) => {
+  return axios({
+    method: "GET",
+    url: `${AT_SCE_SERVICE_URI}/v1/users/${id}`,
+    headers: {
+      "content-type": "application/json",
+    },
+  }).catch(function (error) {
+    console.log(`Error: ${error.message}`);
+  });
```

Comment:
There is already a PR to include this method, can you please remove it?

Reply:
We can continue with this PR, after that we can integrate it with the login